src/__test__/actions/incidenceReports.actions.test.js

```diff
@@ -0,0 +1,59 @@
+// third-party libraries
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+// constants
+import constants from '../../_constants';
+const {
```

Comment:
This should be after all imports. Therefore it should be declared after mock data import